__tests__/data/tiktok_ads_input.json

```diff
@@ -2119,5 +2119,291 @@
         ]
       }
     }
+  },
+  {
+    "message": {
+      "anonymousId": "21e13f4bc7ceddad",
+      "channel": "web",
+      "context": {
+        "app": {
+          "build": "1.0.0",
+          "name": "RudderLabs JavaScript SDK",
+          "namespace": "com.rudderlabs.javascript",
+          "version": "1.0.0"
+        },
+        "library": {
+          "name": "RudderLabs JavaScript SDK",
+          "version": "1.0.0"
+        },
+        "userAgent": "Mozilla/5.0 (platform; rv:geckoversion) Gecko/geckotrail Firefox/firefoxversion",
+        "ip": "13.57.97.131",
+        "locale": "en-US",
+        "os": {
+          "name": "",
+          "version": ""
+        },
+        "screen": {
+          "density": 2
+        },
+        "externalId": [
+          {
+            "type": "tiktokExternalId",
+            "id": "f0e388f53921a51f0bb0fc8a2944109ec188b59172935d8f23020b1614cc44bc"
+          }
+        ]
+      },
+      "messageId": "84e26acc-56a5-4835-8233-591137fca468",
+      "session_id": "3049dc4c-5a95-4ccd-a3e7-d74a7e411f22",
+      "originalTimestamp": "2019-10-14T09:03:17.562Z",
+      "timestamp": "2020-09-17T19:49:27Z",
+      "type": "track",
+      "event": "checkout step completed",
+      "properties": {
```

Comment:
Also update the doc with clear example covering all the properties attributes. Right now `properties` field itself is missing in the example - https://www.rudderstack.com/docs/destinations/streaming-destinations/tiktok-ads/#track

Reply:
[Doc card](https://www.notion.so/rudderstacks/TikTok-Ads-Doc-Update-04e2061458bb476e88b71e3e731ac728)